packages/bits/demo/src/components/demo/wizard-v2/wizard-tooltip/wizard-tooltip-example.component.html

```diff
@@ -0,0 +1,41 @@
+<nui-wizard-horizontal #wizardComponent>
+
+    <!--  content of the 1st step -->
+    <nui-wizard-step-v2 i18n-label label="Very long header text. Very long header text">
+        <!-- content of the current step -->
+        <span i18n>Step with long header text</span>.
+
+        <!-- content of the footer -->
+        <ng-template wizardStepFooter>
```

Comment:
@DmytroPutrin, do you mind creating a ticket to change the selector for this directive to `nuiWizardStepFooter`? It's better to prefix our directives with `nui` to avoid the possibility of naming conflicts on the consumer side. Feel free to change it in the scope of this PR as an alternative to creating a ticket.

Reply:
done